### ISSUE TITLE
Difficulty trying to replicate on the USFM dataset

### ISSUE THREAD

**Issue by zhouly101** [openmedlab/USFM#34]
I can't get the best metric in the paper on the USFM dataset for segmentation tasks. 
Can you give me some advice?

1. How can I fine-tune the model with a gradient learning rate? The default code on github uses only one learning rate to fine-tune the entire model.
2. Can you reveal the best model weights after fine-tuning the USFM dataset for segmentation tasks? I want to try to reproduce the best metric
3. Can you reveal the mask_pre and mask_gt images output by fine-tuning the model on USFM dataset for segmentation tasks?

**Comment by Maoshib** [comment]
请问你复现到了最佳指标了吗，我无法复现出来
